sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs

```diff
@@ -0,0 +1,290 @@
+            Assert.AreEqual(account.Location, account.PrimaryLocation);
+
+            Assert.NotNull(account.PrimaryEndpoints);
+            if (account.Kind != Kind.FileStorage)
+            {
+                Assert.NotNull(account.PrimaryEndpoints.Blob);
+            }
+
+            if (account.Kind == Kind.Storage || account.Kind == Kind.StorageV2)
+            {
+                if (account.Sku.Name != SkuName.StandardZRS && account.Sku.Name != SkuName.PremiumLRS)
+                {
+                    Assert.NotNull(account.PrimaryEndpoints.Queue);
+                    Assert.NotNull(account.PrimaryEndpoints.Table);
+                    Assert.NotNull(account.PrimaryEndpoints.File);
+                }
+
+                if (account.Sku.Name == SkuName.StandardRagzrs)
+                {
+                    Assert.NotNull(account.SecondaryEndpoints.Queue);
+                    Assert.NotNull(account.SecondaryEndpoints.Table);
+                }
+            }
+
+            Assert.AreEqual(Models.ProvisioningState.Succeeded, account.ProvisioningState);
+            Assert.Null(account.LastGeoFailoverTime);
+
+            if (account.Sku.Name == SkuName.StandardLRS || account.Sku.Name == SkuName.StandardZRS || account.Sku.Name == SkuName.PremiumLRS)
+            {
+                Assert.Null(account.SecondaryLocation);
+                Assert.Null(account.StatusOfSecondary);
+                Assert.Null(account.SecondaryEndpoints);
+            }
+            else if (account.Sku.Name == SkuName.StandardGRS)
+            {
+                Assert.AreEqual(AccountStatus.Available, account.StatusOfSecondary);
+                Assert.NotNull(account.SecondaryLocation);
+                Assert.Null(account.SecondaryEndpoints);
+            }
+            else if (account.Sku.Name == SkuName.StandardRagzrs)
+            {
+                Assert.AreEqual(AccountStatus.Available, account.StatusOfSecondary);
+                Assert.NotNull(account.SecondaryLocation);
+                Assert.NotNull(account.SecondaryEndpoints);
+                Assert.NotNull(account.SecondaryEndpoints.Blob);
+            }
+
+            if (useDefaults)
+            {
+                Assert.AreEqual(DefaultLocation, account.Location);
+                Assert.AreEqual(DefaultSkuNameStandardGRS.Name, account.Sku.Name);
+                Assert.AreEqual(SkuTier.Standard, account.Sku.Tier);
+                Assert.AreEqual(DefaultKindStorage, account.Kind);
+
+                Assert.NotNull(account.Tags);
+                Assert.AreEqual(2, account.Tags.Count);
+                Assert.AreEqual("value1", account.Tags["key1"]);
+                Assert.AreEqual("value2", account.Tags["key2"]);
+            }
+        }
+
+        public static AccountSasParameters ParseAccountSASToken(string accountSasToken)
+        {
+            string[] sasProperties = accountSasToken.Substring(1).Split(new char[] { '&' });
+
+            string ServiceParameters = string.Empty;
```

Comment:
local variable names should be camel style

Reply:
Update parameter style.